noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/components/private_call_data_validator.nr

```diff
@@ -356,20 +368,7 @@ impl PrivateCallDataValidator {
             public_inputs.public_call_requests,
             self.array_lengths.public_call_requests,
         );
-
-        let teardown_call_request_count = if public_inputs.public_teardown_call_request.counter == 0
-             {
-                0
-            } else {
-                1
-            };
-            validate_incrementing_counters_within_range(
```

Comment:
Don't need to check it as teardown_call_request does not have a counter anymore.